buildshiprun/handler.go

```diff
@@ -154,13 +154,14 @@ func Handle(req []byte) string {
 			Image:   imageName,
 			Network: "func_functions",
 			Labels: map[string]string{
-				sdk.FunctionLabelPrefix + "git-cloud":      "1",
-				sdk.FunctionLabelPrefix + "git-owner":      event.Owner,
-				sdk.FunctionLabelPrefix + "git-repo":       event.Repository,
-				sdk.FunctionLabelPrefix + "git-deploytime": strconv.FormatInt(time.Now().Unix(), 10), //Unix Epoch string
-				sdk.FunctionLabelPrefix + "git-sha":        event.SHA,
-				"faas_function":                            serviceValue,
-				"app":                                      serviceValue,
+				sdk.FunctionLabelPrefix + "git-cloud":        "1",
+				sdk.FunctionLabelPrefix + "git-owner":        event.Owner,
+				sdk.FunctionLabelPrefix + "git-repo":         event.Repository,
+				sdk.FunctionLabelPrefix + "git-deploytime":   strconv.FormatInt(time.Now().Unix(), 10), //Unix Epoch string
+				sdk.FunctionLabelPrefix + "git-sha":          event.SHA,
+				sdk.FunctionLabelPrefix + "git-private-repo": strconv.FormatBool(event.Private),
```

Comment:
Is this data being populated by `git-tar` yet?

Reply:
cc @martindekov

Reply:
No It is not populated. I can handle the population of the struct in different PR and this one to extend it? But the answer where should we take whether or not is private or not is from payload ```...
  "repository": {
...
    "private": false,
...
``` Like pointed in the issue

Reply:
Also we used 0/1 elsewhere, so I think we should do the same here and not use "true/false"